addons/mail/static/src/models/chatter/chatter.js

```diff
@@ -249,6 +249,9 @@ function factory(dependencies) {
         activitiesState: attr({
             related: 'activities.state',
         }),
+        composer: one2one('mail.composer', {
```

Comment:
Maybe out of scope but
-> `many2one` (multiple chatters should be possible)

`one2one` should be a very special case that is used only when `many2one` has issues.

I think we currently still have too many `one2one` (ex. `thread` here should be `many2one` too).

Reply:
Maybe change thread now too?